src/script/page/useAppState.ts

```diff
@@ -77,21 +76,19 @@ const useAppState = create<AppState>((set, get) => ({
   contentState: ContentState.WATERMARK,
   listState: ListState.CONVERSATIONS,
   previousContentState: null,
-  setContentState: (contentState: ContentState) =>
+  setContentState: (contentState: ContentState) => {
+    const previousContentState = get().contentState;
     set(state => ({
       ...state,
       contentState,
-    })),
+      previousContentState,
+    }));
+  },
```

Comment:
Now setting the previous content is automatically done when setting a new state. 
There was an invisible bug in the implementation that occured when switching to the startUI (that keeps the conversation content on the right side)